pyOCD/core/memory_map.py

```diff
@@ -265,10 +265,10 @@ def getIntersectingRegions(self, start, end=None, length=None, range=None):
     def getXML(self):
         root = ElementTree.Element('memory-map')
         for r in self._regions:
-            mem = ElementTree.SubElement(root, 'memory', type=r.type, start=hex(r.start), length=hex(r.length))
+            mem = ElementTree.SubElement(root, 'memory', type=r.type, start=hex(r.start).rstrip("L"), length=hex(r.length).rstrip("L"))
```

Comment:
According to Python documentation, hex() is only used for int or long. It should be enough for memory address and length.